src/components/CreatePlayer/CreatePlayer.js

```diff
@@ -14,16 +14,18 @@ const CreatePlayer = props => {
   const handlePlayerCreate = player => {
     const newData = [...players];
 
-    newData.push(new Player(player.name, null, player.avatar));
+    newData.push(
+      new Player(player.name, player.sex, null, null, player.avatar)
```

Comment:
At this point, it may be easier for the `Player` to take an object as an argument instead of multiple arguments. That way you don't need `null` values and the order won't matter!